### PR TITLE
extend memoize_zap_cache() to clear all memoized values by default

### PR DESCRIPTION
**Pull request by snarfed**
used in snarfed/color@31a17846f9b70d543a4339954b3b03eaa7ad2302

### CODE REVIEW

**Comment by snarfed** [comment]
@kn i'd assign to you if i could, but i'm not a member of this repo. mind adding me when you get the chance?

**Comment by kn** [comment]
@snarfed added! :) let me know if it still doesn't work.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A memoize decorator.
 
 * Install memoize
 ```
-$ pip install git+https://git+https://github.com/ColorGenomics/memoize.git@v0.1.0
+$ pip install git+https://git+https://github.com/ColorGenomics/memoize.git@v0.1.1
```
Comment (kn) [inline README.md]: let's bump up the minor version since this is a new feature.

**Comment by kn** [comment]
just one comment. o/w LGTM

File: README.md (hunk 1 above)
Comment (snarfed) [inline README.md]: that was fast! sure, done.

**Comment by snarfed** [comment]
thanks! merging.

**Pull request merged**